src/Microsoft.DocAsCode.Common/Git/GitUtility.cs

```diff
@@ -55,45 +57,15 @@ public static GitDetail TryGetFileDetail(string filePath)
 
         public static GitDetail GetFileDetail(string filePath)
         {
-            if (string.IsNullOrEmpty(filePath))
+            if (string.IsNullOrEmpty(filePath) || !ExistGitCommand())
             {
                 return null;
             }
 
-            if (!Path.IsPathRooted(filePath))
-            {
-                throw new GitException($"{nameof(filePath)} should be an absolute path");
-            }
+            var path = Path.Combine(EnvironmentContext.BaseDirectory, filePath).ToNormalizedPath();
```

Comment:
As a generic method, it would be better to avoid using EnvironmentContext from Plugins?

Reply:
It's Environment.CurrentDirectory as default value, I think it is general enough